Update README.md

rZsnWwM5 opened the pull request:
It's more accurate to say that this GitHub repository has been abandoned, because xmikos is now distributing an app with the same name that isn't based on the code in this GitHub repository.

Summarized events leading up to this point.

~~Removed outdated sections that were distracting.~~

Edit: Restored sections as requested.

Commit:
Update README.md

It's more accurate to say that this GitHub repository has been abandoned, because xmikos is now distributing an app with the same name that isn't based on the code in this GitHub repository.

Summarized events leading up to this point.

Removed outdated sections that were distracting.

mimi89999 commented on README.md:
```diff
@@ -1,44 +1,21 @@
+LibreSignal was started in February 2016 so that people using Google-free devices could communicate with people using the official Signal clients. Signal did not support these devices at the time.
 
-In the same thread, Marlinspike said that he would be willing to consider ["a clean, well written, and well tested" pull request that would add WebSocket support to the Android version of Signal](https://github.com/LibreSignal/LibreSignal/issues/37#issuecomment-226646872). This would effectively eliminate Signal's dependency on GCM and thus allow the official Signal app to function on custom Andr ...
+In May 2016, [Open Whisper Systems's stance on federation](https://whispersystems.org/blog/the-ecosystem-is-moving/) meant that LibreSignal users would have had to get their contacts to switch to LibreSignal if the LibreSignal project had set up its own [Signal-Server](https://github.com/WhisperSystems/Signal-Server). When asked if he was OK with LibreSignal using the Open Whisper Systems servers ...
```
First comment and then blog post IIRC.

rZsnWwM5 replied:
If that's an issue, how would you narrate it instead?

rZsnWwM5 replied:
Looking at it again, I think it is probably not that relevant, so I removed it.

mimi89999 replied:
I meant to keep it chronological. The comment is from May 5 and blog post from May 10. So first should be the comment and then the blog post.

rZsnWwM5 replied:
The comment is now first, and then the blog post.

mimi89999 commented on README.md:
```diff
@@ -1,44 +1,21 @@
+In May 2016, [Open Whisper Systems's stance on federation](https://whispersystems.org/blog/the-ecosystem-is-moving/) meant that LibreSignal users would have had to get their contacts to switch to LibreSignal if the LibreSignal project had set up its own [Signal-Server](https://github.com/WhisperSystems/Signal-Server). When asked if he was OK with LibreSignal using the Open Whisper Systems servers ...
 
-To be clear, Signal relying on GCM is only an issue for people who use a custom Android ROM without Google Play Services. For the vast majority of people who do have Google Play on their phone, this issue is completely irrelevant.  Signal is designed so that [GCM is only used for a wakeup event and never sees any of the encrypted Signal messages](http://support.whispersystems.org/hc/en-us/article ...
+Signal has officially supported Google-free devices since February 20, 2017. However, before Open Whisper Systems will officially distribute Signal outside of the Google Play Store, [they would like](https://github.com/WhisperSystems/Signal-Android/commit/1669731329bcc32c84e33035a67a2fc22444c24b#commitcomment-20980219):
```
officially supports....

rZsnWwM5 replied:
Done.

mimi89999 commented on README.md:
```diff
@@ -1,44 +1,21 @@
+>1. A small well-written library that we can link into Signal which does crash reporting, checking for updates (and posting notifications for the user), and reporting basic stats.
+2. A simple web-app that looks like the Play Store console, which allows us to publish new APKs, display crash reports, and display basic stats.
 
-The following text is outdated.
```
Please keep the old readme.

rZsnWwM5 replied:
I have now restored the outdated sections that I thought were distracting.

Commit:
Update README.md

Restored outdated text as requested.

Commit:
Update README.md

has officially supported --> officially supports

Commit:
Update README.md

Open Whisper Systems's stance on federation is perhaps irrelevant, so I removed it.

Commit:
Update README.md

Restored blog post in chronological order as requested.

mimi89999 commented on README.md:
```diff
@@ -1,12 +1,21 @@
-#This project was abandoned because [Moxie Marlinspike said he was not OK with LibreSignal using the Open Whisper Systems servers and the name "Signal"](https://github.com/LibreSignal/LibreSignal/issues/37#issuecomment-217211165).
+#This GitHub repository has been abandoned.
```
I really like to call it **project**
Sorry for being picky.

rZsnWwM5 replied:
>I really like to call it project
>Sorry for being picky.

I think it would be misleading to say that the LibreSignal project has been abandoned, because @xmikos is now distributing an app with the same name that isn't based on the code in this GitHub repository. Everything would be much clearer if @xmikos had chosen a different name for his rebranded version of Signal, but now the LibreSignal project is living on through @xmikos's F-droid repo.

xmikos replied:
Just for the record I have been distributing independent Signal builds named LibreSignal (in my main repository on https://fdroid.eutopia.cz) long before LibreSignal (WebSocket fork) project has been created.

rZsnWwM5 replied:
@xmikos Thanks!  I apologize for getting that the other way around. The timeline for when LibreSignal started was a bit blurry. I have now dug around a bit. Correct me if anything looks incorrect:

* In September 2014, @JavaJens attempted [a pull request](https://github.com/WhisperSystems/Signal-Android/pull/1960) to add websocket support to TextSecure. This was reviewed by Moxie, who [commented that](https://github.com/WhisperSystems/Signal-Android/pull/1960#issuecomment-63234532) the code was not in very good shape and was cut and pasted from third party libraries. JavaJens closed the PR and moved everything to [his own GitHub repo](https://github.com/JavaJens/TextSecure).
* In August 2015, xmikos was already distributing an independent build of TextSecure, but then also [started distributing](https://twitter.com/xmikos/status/636494031464321024) a build that was based on JavaJens's websocket repo, which he initially also called TextSecure. Moxie asked xmikos to rebrand his builds, so xmikos renamed both his independent build and his JavaJens-websocket-build as TextLibre.
* In November 2015, TextSecure and RedPhone merged to become Signal, so xmikos renamed both his independent build and his JavaJens-websocket-build as as LibreSignal.
* In February 2016, mimi89999 [decided](https://github.com/JavaJens/TextSecure/issues/72) to fork JavaJens's GitHub repo to create this LibreSignal GitHub repo, and Xmikos started basing his websocket-build on this repo.

xmikos replied:
I don't remember if dates are exact, but apart from that this timeline is correct.

rZsnWwM5 replied:
@xmikos So when would you say LibreSignal started (since it didn't start in February 2016)? Most people know LibreSignal as the websocket-based build of TextSecure/Signal. Would you agree if I wrote that LibreSignal started [in August 2015](https://twitter.com/xmikos/status/636494031464321024), you started distributing a build that was based on JavaJens's websocket repo?

rZsnWwM5 replied:
@xmikos Never mind, I added the timeline to the README.

mimi89999 commented on README.md:
```diff
@@ -1,12 +1,21 @@
+Signal officially supports Google-free devices as of February 20, 2017. However, before Open Whisper Systems will officially distribute Signal outside of the Google Play Store, [they would like](https://github.com/WhisperSystems/Signal-Android/commit/1669731329bcc32c84e33035a67a2fc22444c24b#commitcomment-20980219):
 
-There are two ways in which you can help make the pull request. If you are able to contribute code, check out [this page about contributing code to Signal-Android](https://github.com/WhisperSystems/Signal-Android/blob/master/CONTRIBUTING.md). If you are not able to contribute code, consider contributing to the aforementioned bounty.
+>1. A small well-written library that we can link into Signal which does crash reporting, checking for updates (and posting notifications for the user), and reporting basic stats.
```
We do have ACRA for crash reporting. https://github.com/ACRA/acra
@moxie0

rZsnWwM5 replied:
>We do have ACRA for crash reporting. https://github.com/ACRA/acra

There's a discussion about this [here](https://whispersystems.discoursehosting.net/t/how-to-get-signal-apks-outside-of-the-google-play-store/808). Do you think we should link to that discussion in the README as well?

Commit:
added timeline for when LibreSignal started

Commit:
Signal is now available as an APK

Removed other recommendations, because Signal is now available as an official APK.

Commit:
Update README.md

added FAQ note
